enterprise/j2ee.common/src/org/netbeans/modules/j2ee/common/dd/DDHelper.java

```diff
@@ -40,273 +43,164 @@
  */
 public class DDHelper {
 
-    private static final String RESOURCE_FOLDER = "/org/netbeans/modules/j2ee/common/dd/resources/"; //NOI18N
+    static final String RESOURCE_FOLDER = "/org/netbeans/modules/j2ee/common/dd/resources/"; //NOI18N
+
+    static Consumer<MakeFileCopy> atomicActionRunner = FileUtil::runAtomicAction;
+
+    static MakeFileCopyFactory makeFileCopyFactory = MakeFileCopy::new;
+
+    private static final List <Profile> WEBXML_REQUIREMENTS_INDEPENDANT_PROFILES = List.of(
+            Profile.JAVA_EE_5,
+            Profile.J2EE_14,
+            Profile.J2EE_13
+    );
```

Comment:
-> EnumSet or Set

Since you are only using `contains()`

Reply:
Replaced with `EnumSet`